tests/integration/README.md

```diff
@@ -0,0 +1,17 @@
+# Integration testing
+
+Test cases are using tox and pytest for running the tests.
+
+## Debug the integration tests
+
+Since integration tests are different with unit test and api test, it requires
+to deploy the latest patch to the openlcs-ci environment. Besides that, when
+manually debug the test cases, **make sure there are no pipeline running**.
+You should always run the commands from OpenLCS root directory.
+
+1. `oc login --server=$OCP_CONSOLE`
```

Comment:
`oc login --token=xxxxxx --server=https://api.mpp-w2-preprod.cfln.p1.openshiftapps.com:6443`

Reply:
The infosec team won't allow any internal URL. I will only add the "--token=xxx", it is acceptable?

Reply:
Yes